demo/main.js

```diff
@@ -303,6 +321,21 @@ Demo.template = xml/* xml */ `
   <div t-else="">
     <Spreadsheet model="model" t-key="state.key"/>
   </div>
+  <dialog
+    class="alert alert-danger"
+    id="error-dialog"
+    t-if="state.errorMessage"
+    open="1"
+  >
+  <div class="d-flex flex-column">
+    <h3 class="text-center">Uncaught Error</h3>
+    <div id="error-dialog-body" class="d-flex flex-column">
+      <t t-foreach="state.errorMessage.split('\n')" t-as="line" t-key="line_index">
+        <span t-esc="line"/><br/>
```

Comment:
The `<br/>` is kinda really ulgy, and are useless since the div is a flex, and will put 1 span/line

Reply:
this cross browser stuff is really getting on my nerves ...
On firefox: 
![image](https://github.com/odoo/o-spreadsheet/assets/10333790/02e52f5a-0220-47c4-a431-c8df0a9960f8)